content/get-started/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-description: Get started with source available Redis
+description: Get started with Redis Community Edition
```

Comment:
After taking a closer look at this page on https://redis.io/docs/latest/get-started/, maybe this should just be "Redis" or changed back to "source available Redis". The first half of the doc seems to be mostly related to Redis CE, but the second half has Enterprise links.
```suggestion
description: Get started with source available Redis
```

Reply:
I think just "Redis" is probably the right idea here.